shadows/framework/src/main/java/org/robolectric/shadows/ShadowWebView.java

```diff
@@ -44,6 +48,12 @@
   private int goBackInvocations = 0;
   private LoadData lastLoadData;
   private LoadDataWithBaseURL lastLoadDataWithBaseURL;
+  private String originalUrl;
+  private List<String> history = new ArrayList<>();
+  private String lastEvaluatedJavascript;
+  // TODO: Delete this when setCanGoBack is deleted. This is only used to determine which "path" we
```

Comment:
Could you fix an issue described in this `TODO` or remove a comment if not actual?